workflow/kfdrc_production_lancet_wf.cwl

```diff
@@ -0,0 +1,187 @@
+
+  # WXS only Fields
+  padded_capture_regions: { type: 'File?', doc: "Recommend 100bp pad, for somatic variant" }
+
+  # Inputs from other solo callers
+  strelka2_vep_vcf: { type: 'File?', doc: "VEP VCF output from Strelka2 caller" }
+  mutect2_vep_vcf: { type: 'File?', doc: "VEP VCF output from Mutect2 caller" }
+
+outputs:
+  lancet_vep_vcf: { type: File, outputSource: run_lancet/lancet_vep_vcf }
+  lancet_vep_tbi: { type: File, outputSource: run_lancet/lancet_vep_tbi }
+  lancet_vep_maf: { type: File, outputSource: run_lancet/lancet_vep_maf }
+  lancet_prepass_vcf: { type: File, outputSource: run_lancet/lancet_prepass_vcf }
+
+steps:
+  choose_defaults:
+    run: ../tools/mode_defaults.cwl
+    in:
+      input_mode: wgs_or_wxs
+      exome_flag: exome_flag
+      lancet_padding: lancet_padding
+      lancet_window: lancet_window
+    out: [out_exome_flag,out_cnvkit_wgs_mode,out_i_flag,out_lancet_padding,out_lancet_window,out_vardict_padding]
+
+  prepare_reference:
+    run: ../sub_workflows/prepare_reference.cwl
+    in:
+      input_fasta: reference_fasta
+      input_fai: reference_fai
+      input_dict: reference_dict
+    out: [indexed_fasta,reference_dict]
+
+  select_interval_list:
+    run: ../tools/mode_selector.cwl
+    in:
+      input_mode: wgs_or_wxs
+      wgs_input: wgs_calling_interval_list
+      wxs_input: padded_capture_regions
+    out: [output]
+
+  gatk_intervallisttools:
+    run: ../tools/gatk_intervallisttool.cwl
+    in:
+      interval_list: select_interval_list/output
+      reference_dict: prepare_reference/reference_dict
+      exome_flag: choose_defaults/out_exome_flag
+      scatter_ct:
+        valueFrom: ${return 50}
+      bands:
+        valueFrom: ${return 80000000}
+    out: [output]
+
+  samtools_cram2bam_plus_calmd_tumor:
+    run: ../tools/samtools_cram2bam_plus_calmd.cwl
+    in:
+      input_reads: input_tumor_aligned
+      threads:
+        valueFrom: ${return 16;}
+      reference: prepare_reference/indexed_fasta
+    out: [bam_file]
+
+  samtools_cram2bam_plus_calmd_normal:
+    run: ../tools/samtools_cram2bam_plus_calmd.cwl
+    in:
+      input_reads: input_normal_aligned
+      threads:
+        valueFrom: ${return 16;}
+      reference: prepare_reference/indexed_fasta
+    out: [bam_file]
+
+  bedops_gen_lancet_intervals:
+    run: ../tools/preprocess_lancet_intervals.cwl
+    in:
+      strelka2_vcf: strelka2_vep_vcf
+      mutect2_vcf: mutect2_vep_vcf
+      ref_bed: lancet_calling_interval_bed
+      output_basename: output_basename
+    out: [run_bed]
+
+  gatk_intervallisttools_exome_plus:
+    run: ../tools/gatk_intervallisttool.cwl
+    in:
+      interval_list: bedops_gen_lancet_intervals/run_bed
+      reference_dict: prepare_reference/reference_dict
+      exome_flag:
+        valueFrom: ${return "Y";}
+      scatter_ct:
+        valueFrom: ${return 50}
+      bands:
+        valueFrom: ${return 80000000}
```

Comment:
no fix necessary, just wanted to document that we checked that `;` is "optional" when terminating the javascript expression and is not passed to the CWL command line